Oye/Fade/Fade/sketch_jul12a/sketch_jul12a.ino

```diff
@@ -0,0 +1,25 @@
+int led = 11;          // the PWM pin the LED is attached to
+int brightness = 0;    // how bright the LED is
+int fadeAmount = 5;    // how many points to fade the LED by
```

Comment:
Well done Oyeeeeeeeeeeeeeeeeeee, you have really smash this Arduino Coding, and its very neat and fabulous to read Oye. And always remember to always add detailed notes to your blink and you will get there. #nevergiveup# #RisequeenOye#  #wickedOyeatcoding# #GirlPower#